app/grandchallenge/core/static/css/core.css

```diff
@@ -25,7 +29,8 @@ p {
 }
 
 /* fix modal display with overflow-y: scroll on html */
-.modal, .modal-open {
+.modal,
```

Comment:
Please undo all of the css formatting changes.

Reply:
Mea Culpa, I did not notice. 

It seems like the basic CSS-language-support for VScode has formatting-on-save turned on by default.

Reply:
Okay we can do this with pre-commit now.